cms/templates/cms/home_page.html

```diff
@@ -68,7 +68,7 @@
         {% endif %}
         <div class="banner-wrapper-content">
           <h1>
-            MITx MicroMasters Programs: Bringing MIT to you
+            MITx MicroMasters&reg; Programs: Bringing MIT to you
```

Comment:
We only need this mark if it helps change the organic search results. Let's remove it for now, and I'll keep an eye on how Google responds. If we need it, we can add it later.